src/tests/ColorPaletteDummy.ts

```diff
@@ -0,0 +1,16 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ColorPalette } from '../types';
+import { ColorDummy } from './ColorDummy';
+
+export class ColorPaletteDummy {
+    static withDefaults(): ColorPalette[] {
+        return [
+            {
+                id: 331,
+                name: 'Example Colors',
+                colors: [ColorDummy.red, ColorDummy.yellow, ColorDummy.green],
+            },
+        ];
```

Comment:
```suggestion
        return [
            {
                id: 331,
                name: 'Example Colors',
                colors: [ColorDummy.red, ColorDummy.yellow, ColorDummy.green],
            },
        ];
```
Can be extracted to singular `ColorPaletteDummy`